brake/backends/cachebe.py

```diff
@@ -51,12 +51,23 @@ def _keys(self, func_name, request, ip=True, field=None, period=None):
 
     def count(self, func_name, request, ip=True, field=None, period=60):
         """Increment counters for all relevant cache_keys given a request."""
-        counters = dict((key, 1) for key in self._keys(
```

Comment:
Yes. Good eye. This definitely would have been a problem. I fixed and added a test case.